lib/application_controller_patch.rb

```diff
@@ -0,0 +1,12 @@
+module ApplicationControllerPatch
+    def self.included(base)
+      base.send(:include, InstanceMethods)
+      base.send(:before_action, :init_users_cache, only: [:show])
```

Comment:
I'm guessing that this is the base `Controller` class and we should add this hash only in `IssuesControlle` and `ForumController`.

What do you think?

Reply:
It worked like you suggested